shelltestrunner.cabal

```diff
@@ -46,7 +46,7 @@ executable shelltest
     base                 >= 4     && < 5,
     Diff                 >= 0.2.0 && < 0.4,
     filemanip            >= 0.3   && < 0.4,
-    HUnit                            < 1.6,
+    HUnit                            < 1.7,
```

Comment:
Allowing HUnit 1.6, I see. Does it compile with HUnit < 1.6 too ? We should probably keep that working. HUnit 1.6 isn't yet widespread, eg stackage LTS has 1.5.

Reply:
Yes it seems fine with older HUnit too.